src/modal/index.tsx

```diff
@@ -103,6 +105,45 @@ const StyledModal = styled(Modal)`
   }
 `
 
+const ModalBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  position: relative;
+`
+
+const ModalFooter = styled.div`
+  position: -webkit-sticky;
+  position: sticky;
+  bottom: 0;
+  background-color: ${color.white};
+  flex-shrink: 0;
+  z-index: 2;
+
+  .kirk-modal-modalFooterBorder & {
```

Comment:
How will this class be toggled?

Reply:
This class is toggled based on the `showFooterBorder` state in the Modal. Without a bigger refacto (and probably breaking changes) that's how far I could get.
1. Added the `onScroll` event on Modal scrollable content
2. When reaching the bottom we update the `showFooterBorder` state which activate the `kirk-modal-modalFooterBorder` class
3. `ModalFooter` then applies its border based on `kirk-modal-modalFooterBorder`

Reply:
Oh! I didn't see the `&` at the end of the selector. This changes everything 👍